src/lib/metrics.ts

```diff
@@ -285,12 +285,18 @@ export default class MetricsMonitor {
             help: 'Duration of feature lifecycle stages',
         });
 
-        const stageCountByProject = createGauge({
-            name: 'stage_count_by_project',
+        const featureLifecycleStageCountByProject = createGauge({
+            name: 'feature_lifecycle_stage_count_by_project',
```

Comment:
this will required prometheus query update so I want to do it today because we introduced this metrics only today